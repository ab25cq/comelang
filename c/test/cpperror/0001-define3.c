/*
PATTERN: garbage
PATTERN: c:7:
*/

#define X
#undef X asdfad


