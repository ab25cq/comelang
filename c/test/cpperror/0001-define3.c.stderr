garbage at end of #undef at test/cpperror/0001-define3.c:7:10
#undef X asdfad
         ^
